### ISSUE TITLE
Add a script that can post an image to Twitter

### ISSUE THREAD

**Issue by vladiliescu** [vladiliescu/DeceseCOVID#4]
A script/function - it should be able to receive the path to an image (e.g. `./out.png`), and be able to post it automatically to a Twitter account, using a predefined text/alt text.
